.github/ISSUE_TEMPLATE/github-request---access.yml

```diff
@@ -1,13 +1,16 @@
-name: GitHub Request - Generic
-description: Make any changes to the openedx organization (grant/revoke user access, change config, etc).
+name: GitHub Access/Config Request
```

Comment:
To be consistent with the `GitHub Request - <TYPE>` pattern of the other requests, how about calling this `GitHub Request - Access/Config`?

Alternatively, I'd also support changing them all to `GitHub <TYPE> Request` to match what you have here.